examples/AWSDriverExample/src/main/java/software/amazon/MetricsExample.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon;
+
+import com.amazonaws.xray.AWSXRay;
+import com.amazonaws.xray.AWSXRayRecorderBuilder;
+import com.amazonaws.xray.entities.Segment;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
+import io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter;
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import software.amazon.jdbc.PropertyDefinition;
+
+public class MetricsExample {
+
+  // User configures connection properties here
+  public static final String POSTGRESQL_CONNECTION_STRING =
+      "jdbc:aws-wrapper:postgresql://atlas-postgres.cluster-czygpppufgy4.us-east-2.rds.amazonaws.com:5432/postgres";
+  private static final String USERNAME = "pgadmin";
+  private static final String PASSWORD = "my_password_2020";
+
+  private static final String SQL_DBLIST = "select datname from pg_database;";
+  private static final String SQL_SLEEP = "select pg_sleep(20);";
+  private static final String SQL_TABLELIST = "select * from information_schema.tables where table_schema='public';";
+
+  private final OpenTelemetry openTelemetry;
+  private final OtlpGrpcSpanExporter spanExporter;
+  private final OtlpGrpcMetricExporter metricExporter;
+  private final SdkTracerProvider tracerProvider;
+  private final SdkMeterProvider meterProvider;
+
+  public MetricsExample() {
```

Comment:
I'm still confused by code in constructor. It seems it configure open telemetry and at the same time I see another code in main() that configure XRay. I believe we need to simplify this example.

Reply:
Now i understand. I was using one example class to test everything, just switching parameters.
Will split the examples.